corehq/apps/geospatial/templates/geospatial/partials/index_alert.html

```diff
@@ -0,0 +1,28 @@
+{% load i18n %}
+
+{% if task_status.status == 'ERROR' %}
+<div class="alert alert-danger">
+  <p>
+    {% if task_status.error_slug == 'TOO_MANY_CASES' %}
+      {% blocktrans %}
+        This domain contains too many cases and so they will not be made available
```

Comment:
nit:

I'd suggest reframing that message a bit.
I read this and it appeared to me that the feature won't work at all of the project space though I believe what we want to say is that the existing cases will not such in microplanning reports but any new/updated cases will.

Reply:
That sounds reasonable, I can see how this might come across as confusing for users. 

How does the following phrasing sound?
```
This domain contains too many cases, and so all existing cases will not be
available in any of the reports for this feature. New or updated cases will
still be available for use by this feature. Please reach out to support.
```

Reply:
Made slight changes

Existing cases in the project were not processed to be available in Microplanning reports because they were too many to be processed. New or updated cases will still be available for use for Microplanning. Please reach out to support if you need support with existing cases.